[2024-03-02T18:57:07.726+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_ncei_data.download_html_data scheduled__2024-01-04T00:00:00+00:00 [queued]>
[2024-03-02T18:57:07.743+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_ncei_data.download_html_data scheduled__2024-01-04T00:00:00+00:00 [queued]>
[2024-03-02T18:57:07.744+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2024-03-02T18:57:07.886+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_html_data> on 2024-01-04 00:00:00+00:00
[2024-03-02T18:57:07.895+0000] {standard_task_runner.py:60} INFO - Started process 352 to run task
[2024-03-02T18:57:07.897+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_ncei_data', 'download_html_data', 'scheduled__2024-01-04T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/hi.py', '--cfg-path', '/tmp/tmpii36y_vt']
[2024-03-02T18:57:07.899+0000] {standard_task_runner.py:88} INFO - Job 33: Subtask download_html_data
[2024-03-02T18:57:08.097+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_ncei_data.download_html_data scheduled__2024-01-04T00:00:00+00:00 [running]> on host a890577b40a7
[2024-03-02T18:57:08.330+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='fetch_ncei_data' AIRFLOW_CTX_TASK_ID='download_html_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-04T00:00:00+00:00'
[2024-03-02T18:57:08.332+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-02T18:57:08.334+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "curl {{ dag_run.conf.get('base_url', params.base_url) }}2001/ --create-dirs -o /tmp/html/2001.html"]
[2024-03-02T18:57:08.346+0000] {subprocess.py:86} INFO - Output:
[2024-03-02T18:57:08.348+0000] {subprocess.py:93} INFO - /usr/bin/bash: -c: line 1: syntax error near unexpected token `('
[2024-03-02T18:57:08.348+0000] {subprocess.py:93} INFO - /usr/bin/bash: -c: line 1: `curl {{ dag_run.conf.get('base_url', params.base_url) }}2001/ --create-dirs -o /tmp/html/2001.html'
[2024-03-02T18:57:08.349+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-03-02T18:57:08.363+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-03-02T18:57:08.367+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=fetch_ncei_data, task_id=download_html_data, execution_date=20240104T000000, start_date=20240302T185707, end_date=20240302T185708
[2024-03-02T18:57:08.491+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 33 for task download_html_data (Bash command failed. The command returned a non-zero exit code 2.; 352)
[2024-03-02T18:57:08.514+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-02T18:57:08.551+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
